tests/conftest.py

```diff
@@ -683,26 +683,123 @@ def product_level_constant_forecast_w_quantiles(hierarchical_structure):
             "timestamp": ["2000-01-05", "2000-01-06"] * 4,
             "segment": ["a"] * 2 + ["b"] * 2 + ["c"] * 2 + ["d"] * 2,
             "target": [1, 1] + [2, 2] + [3, 3] + [4, 4],
-            "target_0.25": [1, 1] + [2, 2] + [3, 3] + [4, 4],
-            "target_0.75": [1, 1] + [2, 2] + [3, 3] + [4, 4],
-        }
+            "target_0.25": [1 / 2, 1 / 4] + [1, 1 / 2] + [2, 1] + [3, 2],
```

Comment:
Why do we need this change?

Reply:
To make sure that transformations are correct and we do't shuffle the columns values